lib/src/client.dart

```diff
@@ -2295,6 +2347,16 @@ class Client extends MatrixApi {
               content: update.content)));
         }
       }
+
+      // Any kind of member change? We should invalidate the profile then:
```

Comment:
Hm, could we just refresh the cache instead of marking it as outdated? i.e can you list the reasons for not doing so?

Reply:
I've listed the reason